ui/src/pages/OBProxy/Detail/Overview/DetailConfig.tsx

```diff
@@ -93,8 +91,16 @@ export default function DetailConfig({
             defaultMessage: '参数设置',
           })}
         </h3>
-        <pre>{JSON.stringify(parameters)}</pre>
+        <InputLabelComp allowDelete={false} disable={true} value={parameters || []} />
       </div>
+      {props.name && props.namespace ? (
+        <ConfigDrawer
+          open={drawerOpen}
+          onClose={() => setDrawerOpen(false)}
+          width={880}
+          {...props}
```

Comment:
Do `ConfigDrawer` and `DetailConfig` share the same prop type? If so, declare the common type once.